src/bin-install.ts

```diff
@@ -14,15 +14,21 @@ typings install [<name>=]<location>
   <location>  The location to read from (described below)
 
 Valid Locations:
-  [<source>!]<pkg>[@<version>][#<tag>]
+  <pkg>[@<version>][#<tag>]
+    Install from "defaultSource" specified in `.typingsrc`
+      default to "npm" for non-ambient package
+      default to "dt" for ambient package
+  <source>!<pkg>[@<version>][#<tag>]
+    Install from registry mirror or DefinitelyTyped (see <source> below)
```

Comment:
In general, agree.

However many people coming from DT. So the dilemma is:
1. We try to be explicit and move dt away from `typings install <pkg> --ambient` syntax. We currently did that by having `typings install dt!<pkg>` but setting defaultAmbientSource = dt.
2. We like to avoid calling out DT so explicitly. But (1) is trying to make it explicit....